OpenRA.Mods.Common/Activities/Move/MoveAdjacentTo.cs

```diff
@@ -161,7 +161,7 @@ public override IEnumerable<Target> GetTargets(Actor self)
 			return Target.None;
 		}
 
-		public override bool Cancel(Actor self)
+		public override bool Cancel(Actor self, bool keepQueue = false)
```

Comment:
No, not really. I kept the flag (ie. pass it on to `base.Cancel()` in those cases where the activity does not use the next activity to do more of its work (like `Move`, `Enter` and `PickupUnit` do, for example). In those other cases, I did not pass it on to keep the current behavior and not introduce bugs. I went through the list of changes and made sure they were consistent with this, hence the update.

This will all be cleaned up when those activities get rewritten.